docs/getting-started-with-scalardb.md

```diff
@@ -144,9 +145,10 @@ $ ../../gradlew run --args="-mode storage -action charge -amount 0 -to merchant1
 $ ../../gradlew run --args="-mode storage -action pay -amount 100 -to merchant1 -from user1"
 ```
 
-## Set up database schema with transaction
+## Set up database schema for transaction
 
 To apply transaction, we can just add a key `transaction` and value as `true` in Scalar DB scheme.
+You can modify the JSON file `emoney-storage.json` by the json object bellow.
```

Comment:
```suggestion
You can create the JSON file `emoney-transaction.json` with the JSON below.
```
We are using `emoney-transaction.json` below so I think we should mention the name.